src/Tribe/Attendees_Table.php

```diff
@@ -70,7 +70,15 @@ public function __construct( $args = [] ) {
 
 		// Fetch the event Object
 		if ( ! empty( $_GET['event_id'] ) ) {
-			$this->event = get_post( absint( $_GET['event_id'] ) );
+			/**
+			 * This filter allows retrieval of an event ID to be filtered before being accessed elsewhere.
+			 *
+			 * @since TBD
+			 *
+			 * @param int The event ID to be filtered.
+			 */
+			$event_id = apply_filters( 'tec_tickets_filter_event_id', $_GET['event_id'] );
```

Comment:
We have multiple places where we apply this filter, to avoid problems in the future with which params are being set changing for one but not the others, do you mind creating a method for this?

Reply:
This is a good chance to remove the direct access to the `GET` superglobal and use some `filter_var` to sanitize to an integer value.

Reply:
The `TEC\Tickets\Event` class is a good place to put the method wrapping the application of the `tec_tickets_filter_event_id` filter. Something like this:

```php
public static function get_event_id( $event_id ): ?int;
```